client/components/edit/ObjectButtonBar.tsx

```diff
@@ -19,7 +19,7 @@ const ObjectButtonBar = ({ pid }: ObjectButtonBarProps): React.ReactElement => {
             <ObjectStatus pid={pid} />
             <EditParentsButton pid={pid} />
             <button onClick={() => clearPidFromChildListStorage(pid)}>
-                <Refresh style={{ height: "14px" }} titleAccess="Refresh children" />
+                <Refresh style={{ height: "20px", verticalAlign: "sub" }} titleAccess="Refresh children" />
```

Comment:
Should we give this a class and move the styling to CSS? I'm pretty sure the direct style here was just my laziness, but if we're going to get fancier, maybe we should do things "the right way."